omero_search_engine/validation/results_validator.py

```diff
@@ -747,14 +1027,26 @@ def get_no_images_sql_containers():
             messages.append(
                 "\n-----------------------------------------------------------------------------\n"  # noqa
             )
-    base_folder = "/etc/searchengine/"
-    if not os.path.isdir(base_folder):
-        base_folder = os.path.expanduser("~")
+    if write_report:
+        base_folder = "/etc/searchengine/"
+        if not os.path.isdir(base_folder):
+            base_folder = os.path.expanduser("~")
+        report_file = os.path.join(base_folder, "check_containers_report.txt")
+        report = "\n".join(messages)  # noqa
+        with open(report_file, "w") as f:
+            f.write(report)
 
-    report_file = os.path.join(base_folder, "check_containers_report.txt")
-    report = "\n".join(messages)  # noqa
-    with open(report_file, "w") as f:
-        f.write(report)
+
+"""
+def set_owner_ship(resource , name, value, owener_id=None, group_id=None):
```

Comment:
rename method to 
``set_ownership`` 
Also the parameter should be ``owner_id``